components/sections/abc/content.en-GB.md

```diff
@@ -66,7 +66,7 @@ And that's it, all values found.
 
 <div className="note">
 
-Much like the *u(t)* function in the above note, the *ratio(t)* function depends on whether we're looking at quadratic or cubic curves. Their form is intrinsically related to the *u(t)* function in that they both come rolling out of the same function evalution, explained over on [MathOverflow](http://mathoverflow.net/questions/122257/finding-the-formula-for-Bézier-curve-ratios-hull-point-point-baseline) by Boris Zbarsky and myself. The ratio functions are the "s(t)" functions from the answers there, while the "u(t)" functions have the same name both here and on MathOverflow.
+Much like the *u(t)* function in the above note, the *ratio(t)* function depends on whether we're looking at quadratic or cubic curves. Their form is intrinsically related to the *u(t)* function in that they both come rolling out of the same function evaluation, explained over on [MathOverflow](http://mathoverflow.net/questions/122257/finding-the-formula-for-Bézier-curve-ratios-hull-point-point-baseline) by Boris Zbarsky and myself. The ratio functions are the "s(t)" functions from the answers there, while the "u(t)" functions have the same name both here and on MathOverflow.
```

Comment:
I have no idea what github is saying here - there does not appear to be a difference.

Reply:
That one's evalution -> evaluation

It's easier to see in github's "rich diff".

Reply:
Where would I find github's "rich diff"? I only know about the unified vs. split view, and neither of those shows the actual different.

Reply:
Do you see a document icon in the toolbar above each diff? That's the rich diff button (yeah, it's not obvious).

Reply:
dear lord, you need to click that for every file? That's kind of ridiculous. Thanks for pointing out it exists at least.